PhotoEnhancer/PhotoEnhancer/Filters/NoiseFilter.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Drawing;
+using System.Windows.Forms;
+using System.Collections.Generic;
+
+namespace PhotoEnhancer
+{
+    public class NoiseFilter : PixelFilter<NoiseParameters>
+    {
+        public override string ToString()
+        {
+            return "Интенсивность шума";
+        }
+
+        public override Pixel ProcessPixel(Pixel originalPixel,
+           NoiseParameters parameters)
+        {
+            var red = originalPixel.R;
+            var green = originalPixel.G;
+            var blue = originalPixel.B;
```

Comment:
Зачем вы присваиваете значения эти переменным, если вы далее не используете эти переменные в правых частях операторов присваивания? Достаточно декларации.
```suggestion
           double red, green, blue;
```